src/info/repo/description.rs

```diff
@@ -21,7 +24,24 @@ impl InfoField for DescriptionInfo {
 
     fn value(&self) -> String {
         match &self.description {
-            Some(v) => v.into(),
+            Some(value) => {
+                if !value.is_empty() && value.contains(' ') {
```

Comment:
```suggestion
                if value.contains(' ') {
```
Just because I'm obsessed with micro-optimizations :laughing:
I'm pretty sure a string would never be empty if it contains `' '`.

Reply:
🤦